### PR TITLE
Adding contributions from Yuri Lin ORCID 0009-0006-9784-2694.

### PR DESCRIPTION
**Pull request by zivy**
Adding reagent information contributed by Yuri Lin.

### CODE REVIEW

File: docs/supporting_material/CD15_AF532/Human_Tonsil_FFPE_Hoechst_blue_CD15_red.jpg
Comment (radtkea) [inline docs/supporting_material/CD15_AF532/Human_Tonsil_FFPE_Hoechst_blue_CD15_red.jpg:1]: This does not look like the right image for CD15. This looks like it labels all cells. Is this CD45 and not CD15?

**Review by radtkea — COMMENTED** [review]
Thanks Ziv! The CD15 image does not look correct. Looks like a different marker?

File: docs/supporting_material/CD15_AF532/Human_Tonsil_FFPE_Hoechst_blue_CD15_red.jpg
Comment (zivy) [reply docs/supporting_material/CD15_AF532/Human_Tonsil_FFPE_Hoechst_blue_CD15_red.jpg:1]: Agreed. CD15 entry was removed from the contribution.

**Review by radtkea — APPROVED** [review]
Looks good to me! Thank you so much:)

**Pull request merged**